src/types/generated/index.ts

```diff
@@ -0,0 +1,11 @@
+/* GENERATED BY https://github.com/Birkbjo/dhis2-open-api-ts */
+
+export type * from './models'
+export type {
+    GistModel,
+    GetReferencedModels,
+    PickValue,
+    PickReferenceProperties,
+    GetReferencedModelsUnion,
+    PickInModelReferences,
+} from './utility'
```

Comment:
Why not `export * from './utility'`?

Reply:
I can change it to *. It's just been through some iterations. Earlier the index had named exports for each model. So I just followed that pattern. By merging to one file it was easier to just `import *`. The only benefit from named export that I can think of is that it makes it easier to `rename` a single export.

Reply:
I guess in this case it's easier to use `export *` as that will allow us to generate types again without having to touch this file